src/App.tsx

```diff
@@ -15,36 +15,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Default = 'default',
+  Alphabetically = 'alphabetically',
+  Length = 'length',
+}
+
 export const App: React.FC = () => {
+  const [sortMode, setSortMode] = useState<SortType>(SortType.Default);
+  const [isReverse, setIsReverse] = useState<boolean>(false);
+  const sorteredGoods: string[] = [...goodsFromServer].sort((a, b) => {
```

Comment:
The variable `sorteredGoods` is misspelled. It should be `sortedGoods` to reflect its purpose more accurately.

Reply:
> The variable `sorteredGoods` is misspelled. It should be `sortedGoods` to reflect its purpose more accurately.

Good point